packages/client/src/comp/WarningMsg.tsx

```diff
@@ -33,13 +34,15 @@ const FloatingButton = styled(Button)`
   border-top-left-radius: 4px;
   /* One above the modal */
   z-index: 31;
+  transition: width ease .2s;
```

Comment:
Let's make the animation go to the bottom right where the red box is.  That way, if users dismiss the page, they see it go to the bottom right.